uco-master/uco.ttl

```diff
@@ -12,14 +12,17 @@
 # imports: http://unifiedcyberontology.org/types
 # imports: http://unifiedcyberontology.org/victim
 
-@base <http://unifiedcyberontology.org/master> .
+@base <http://unifiedcyberontology.org/uco> .
+@prefix : <http://unifiedcyberontology.org/uco#> .
```

Comment:
I think this one is extraneous

Reply:
@sbarnum This is due to our adding the uco-master directory and creating an ontology file that imports everything. It is not the uco-core/#core ontology.  

So, I actually think it is correct.

Reply:
It is good to go.

Reply:
looks good